glean-core/ios/Glean/Glean.swift

```diff
@@ -570,5 +587,7 @@ public class Glean {
         // Init Glean.
         testDestroyGleanHandle()
         initialize(uploadEnabled: uploadEnabled, configuration: configuration)
+        // Enable ping logging for all tests
+        setLogPings(true)
```

Comment:
if you set this after `Glean.initialize` then we might miss logging the pings generated in `initialize`